ui/src/config/section/config.js

```diff
@@ -19,7 +19,7 @@ export default {
   name: 'config',
   title: 'label.configuration',
   icon: 'setting',
-  permission: ['listConfigurations'],
+  permission: ['listConfigurations', 'listInfrastructure'],
```

Comment:
Adding this so the global settings tab doesn't show to domain admins since they now have access to `listConfigurations`